### PR TITLE
More times

### LINKED ISSUE #21: tdate as epoch
tdate is a reasonable `tagged-epoch-id` given Evidence contains a timestamp in `tdate` format and a reference value is expressed in terms of a grace period within which the timestamp falls, relative to the epoch id.

### PR DESCRIPTION
**Pull request by thomas-fossati**
fixes #21

### CODE REVIEW

File: draft-birkholz-rats-epoch-markers.md [hunk 1]
```diff
@@ -151,12 +151,12 @@ At the top level, an Epoch Marker is a CBOR array with a header carrying an opti
 
 This memo comes with a set of predefined payloads.
 
-### CBOR Time Tag (etime)
+### CBOR Time Tags
 
-CBOR extended time tag (1001) optionally bundled with a nonce.
+A CBOR time representation choosing from #6.0/tdate (ISO 8601), #6.1/time (IEEE 1003-1.2017 Seconds since Epoch), or #6.1001/etime, and option bundled with a nonce.
```
Comment (thomas-fossati) [inline draft-birkholz-rats-epoch-markers.md:156]:
```suggestion
A CBOR time representation choosing from CBOR tag 0 (RFC3339 time as a string), tag 1 (Posix time as int or float) or tag 1001    "extended time", optionally bundled with a nonce.
```

**Comment by henkbirkholz** [comment]
fixes #21

**Review by thomas-fossati — COMMENTED** [review]
LGTM (I cannot approve this explicitly because I was the one creating it...)

File: draft-birkholz-rats-epoch-markers.md [hunk 2]
```diff
@@ -151,12 +151,12 @@ At the top level, an Epoch Marker is a CBOR array with a header carrying an opti
...
 
-### CBOR Time Tag (etime)
+### CBOR Time Tags
 
-CBOR extended time tag (1001) optionally bundled with a nonce.
+A CBOR time representation choosing from CBOR tag 0 (tdate, RFC3339 time as a string), tag 1 (time, Posix time as int or float) or tag 1001 (extended time data item), optionally bundled with a nonce.
 
 See {{Section 3 of -CBOR-ETIME}} for the (many) details about the CBOR extended
-time format.
+time format (tag 1001). See {{-CBOR}} for tdate (tag 0) and time (tag 1).
```
Comment (cabo) [inline draft-birkholz-rats-epoch-markers.md:159]: Do we say that "etime" is defined in -CBOR-ETIME?  Shouldn't we?  Is it defined elsewhere?

**Review by cabo — APPROVED** [review]
... but see comment.

File: draft-birkholz-rats-epoch-markers.md (hunk 2 above)
Comment (cabo) [reply draft-birkholz-rats-epoch-markers.md:159]: Actually, -CBOR-ETIME defines `Etime`, and `etime`  is defined here.  WFM.

Comment (henkbirkholz) [reply draft-birkholz-rats-epoch-markers.md:159]: ack

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**